pyproject.toml

```diff
@@ -35,6 +44,14 @@ psycopg2-binary = "^2.9.5"
 djangorestframework = "^3.14.0"
 drf-spectacular = "^0.26.0"
 django-filter = "^22.1"
+django-filter-stubs = "^0.1.2"
+djangorestframework-stubs = "^1.10.0"
+django-stubs = "^1.16.0"
```

Comment:
are you sure these stubs belong here, not in `dev` group?

Reply:
They are definitely `dev`. fixed